example/README.md

```diff
@@ -73,7 +74,7 @@ rust: rustc 1.63.0-nightly
 
     ```bash
     # for m1:
-    cargo install mrepl --version '0.18.0' --root ~/.fluence/cargo
+    cargo install mrepl --version '0.18.6' --root ~/.fluence/cargo
     ```
```

Comment:
currently, that has no impact from a version perspective. whatever fluence cli specifies, it looks for. so there needs to be a way for the user to figure out what fluence cli wants before they manually install.